src/messages.rs

```diff
@@ -0,0 +1,79 @@
+use libs::time::Duration;
+use std::convert::TryInto;
+use std::time::Instant;
+
+use errors::Error;
+use site::Site;
+
+/// Display in the console the number of pages/sections in the site
+pub fn notify_site_size(site: &Site) {
```

Comment:
This messages.rs module contains the stuff from `src/console.rs` that's about formatting log messages.  The colorization stuff moved to a new crate: `components/console`